lib/fixtures/changelog_template.md.erb

```diff
@@ -0,0 +1,11 @@
+# Title
+
+Header 
+
+<% changes.each do |version_data| %>
```

Comment:
@felipe-michelon It's really difficult to intentify from where this ```changes``` object comes from. You should white a comment here or where you call the ```ERB.new``` method.